test/fixtures/utf8-skip-escape-characters.json

```diff
@@ -0,0 +1,61 @@
+{
+    "charset": "utf-8",
+    "headers": {
```

Comment:
Can you please remove these headers (except Content-Type)?
I know some test fixtures have them, but more recent ones don't. Having short files makes it so much easier to find issues in tests. Thanks!

Reply:
I simplified it a bit but some stuff were needed